tests/feature/karma/feature_test.go

```diff
@@ -0,0 +1,46 @@
+package karma
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/jakevoytko/crbot/app"
+	"github.com/jakevoytko/crbot/feature/karma"
+	"github.com/jakevoytko/crbot/testutil"
+)
+
```

Comment:
I can't think of a compelling reason to add any helper functions to testutil/runner. That being said, my testing feels sparse, and there may be a corner case I'm missing.

Reply:
yeah, typically I like to assert the state in storage. But I think this use case is so simple that you don't need to.